components/browser/state/src/main/java/mozilla/components/browser/state/reducer/TabListReducer.kt

```diff
@@ -62,7 +63,16 @@ internal object TabListReducer {
             }
 
             is TabListAction.SelectTabAction -> {
-                state.copy(selectedTabId = action.tabId)
+
+                val updatedTabsList = state.tabs.map {
```

Comment:
nit: I think we could use instead ` state.findTab(action.tabId)` :)

Reply:
I also thought this would work, but since we're updating the `tabs` for the `BrowserState`, we need to map the timestamp to the selected tab in the tab list, and then `copy` that over.